trail/upload/process_uploads/standardizers/whipple_standardizer.py

```diff
@@ -0,0 +1,55 @@
+"""
+Class that facilitates header metadata translation for Las Cumbres Observatory.
+"""
+
+from datetime import datetime, timezone
+
+from upload.process_uploads.header_standardizer import HeaderStandardizer
+from upload.models import Metadata
+
+
+__all__ = ["WhippleStandardizer", ]
+
+
+class WhippleStandardizer(HeaderStandardizer):
+
+    name = "whipple_standardizer"
+    priority = 1
+
+    def __init__(self, header, **kwargs):
+        super().__init__(header, **kwargs)
+
+    @classmethod
+    def canStandardize(self, header, **kwargs):
+        observat = header.get("OBSERVAT", False)
+        if observat and "WHIPPLE" in observat.upper():
+            return True
+        return False
+
+    def standardizeMetadata(self):
+        startt = datetime.strptime(self.header["DATE-OBS"], "%Y-%m-%dT%H:%M:%S.%f%z")
+        endt = datetime.strptime(self.header["DATE-END"], "%Y-%m-%dT%H:%M:%S.%f%z")
+        startt = startt.astimezone(timezone.utc)
+        endt = endt.astimezone(timezone.utc)
+
+        # TODO: there are multiple different telescopes
+        # at the Whipple Obs site, check if there are any particularities
+        # (height specifically) and implement conditions to handle them
+        # http://www.sao.arizona.edu/FLWO/whipple.html
+        # This is the Ridge location height
+        height = 2340
+
+        meta = Metadata(
+            obs_lon=self.header["LONGITUD"],
```

Comment:
Field is definitely truncated `LONGITUD` not `LONGITUDE`? Just checking

Reply:
Yep, 8 char limit for fits header keys in general, I think, but def `LONGITUD` in this one.